plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.block_theme.spec.ts

```diff
@@ -393,6 +393,44 @@ test.describe( 'Product Collection', () => {
 					sidebarSettings.getByLabel( SELECTORS.onSaleControlLabel )
 				).toBeChecked();
 			} );
+
+			test( 'is enlabled by default in 1st Product Collection and disabled in 2nd+', async ( {
```

Comment:
I believe there is a small typo in the test title

```suggestion
			test( 'is enabled by default in 1st Product Collection and disabled in 2nd+', async ( {
```